docs/domains/Domain.md

```diff
@@ -225,6 +226,7 @@ ServerPod describes the configuration for a Kubernetes pod for a server.
 | --- | --- | --- |
 | `enabled` | Boolean | True, if this domain is deployed under an Istio service mesh. Defaults to true when the 'istio' element is included. Not required. |
 | `readinessPort` | number | The WebLogic readiness port for Istio. Defaults to 8888. Not required. |
+| `envoyPort` | number | The envoy port for Istio. Defaults to 31111. Not required. |
```

Comment:
Let's ping Mark on this today... I think we should kill this.

Reply:
removed